src/Component/InlineEdit.php

```diff
@@ -0,0 +1,179 @@
+<?php
+/**
+ * A Simple inline editable text Vue component.
+ */
+
+namespace atk4\ui\Component;
+
+use atk4\ui\Exception;
+use atk4\ui\jsVueService;
+use atk4\ui\View;
+
+class InlineEdit extends View
+{
+    public $defaultTemplate = 'inline-edit.html';
+
+    /**
+     * jsCallback for saving data.
+     *
+     * @var null
+     */
+    public $cb = null;
+
+    /**
+     * Whether callback should save value to db automatically or not.
+     * Default to using onChange handler.
+     * If set to true, then saving to db will be done when model get set
+     * and if model is loaded already.
+     *
+     * @var bool
+     */
+    public $autoSave = false;
```

Comment:
Perhaps the name of the callback should be "saveCallback" ? I'm also confused if this is same as saveOnBlur..  Maybe a simplification is needed?

Reply:
Add some explanation.